app/views/categories/index.html.erb

```diff
@@ -0,0 +1,92 @@
+<div class="header">
+  <%= link_to "<i class='fas fa-arrow-left'></i>".html_safe, :back, class: "btn" %>
+  <h1>Categories</h1>
+  <%= form_for(current_user, url: destroy_user_session_path, method: :delete, html: { class: "nav-link" }) do |f| %>
+    <%= f.button type: :submit, class: "btn-logout" do %>
+      <%= content_tag :i, "", class: "fa-solid fa-right-from-bracket" %>
+    <% end %>
+  <% end %>
+</div>
+<% if @categories.any? %>
+  <ul class="categories">
+    <% @categories.each do |category| %>
+      <li class="category">
+        <% if category.icon.present? %>
+          <%= raw category.icon %>
+        <% end %>
+        <h3><%= category.name %>:</h3>
+        <p>Total: $<%= category.expenses.collect(&:amount).sum %></p>
+        <div class="btn">
+          <button><a href="<%= category_expenses_path(category) %>">View Expenses</a></button>
+          <button><%= link_to 'Add a New Expense', new_category_expense_path(category) %></button>
+        </div>
+      </li>
+    <% end %>
+  </ul>
+  <button class="btn"><%= link_to 'Add a New Category', new_category_path %></button>
+<% else %>
+  <p class="msg">No categories found. Click the button below to add a new category.</p>
+  <button class="btn"><%= link_to 'Add a New Category', new_category_path%></button>
+<% end %>
+
+<style>
+  .header {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
```

Comment:
- [ ] I would suggest that you style your app by creating an external stylesheet. Using an external stylesheet gives you the option to reuse your code. This practice ensures that your code is clean.

- [ ] On the header class, I would advise that you remove the width and adjust the padding to align with the provided style guide.

Reply:
Concerning the styling, I've tried to style the application in an external file in the assets folder, but the thing is, the styling wasn't applying to the views, so the only solution was to style every single view alone.

Reply:
I'm gonna make sure to  remove the width and adjust the padding to align with the provided style guide